controllers/user.js

```diff
@@ -1,13 +1,14 @@
 const UserModel = require('../models/user')
 
 module.exports.get = async function get (req, res) {
-  const data = await UserModel.find({})
-  res.json({data})
+  const data = await UserModel.find({}).populate('office')
+  if (res.locals.role === 'admin') res.json({data})
+  else res.status(401).send('You do not have enough privileges for see this')
 }
 
 module.exports.getOne = async function getOne (req, res) {
-  const data = await UserModel.findById(req.params.id)
-  if (!data) return res.status(200).send('This id is not stored in the server')
+  const data = await UserModel.findById(req.params.id).populate('office')
+  if (!data) return res.status(200).send('This user is not stored in the server')
```

Comment:
status 200 is the default stats, you dont need to call status() in this case

Reply:
Got it, thanks!